flink-table-store-connector/src/test/java/org/apache/flink/table/store/connector/source/TestDataReadWrite.java

```diff
@@ -75,20 +77,19 @@ public FileStoreRead createRead() {
     }
 
     public List<SstFileMeta> writeFiles(
-            BinaryRowData partition, int bucket, List<Tuple2<Integer, Integer>> kvs)
-            throws Exception {
+            BinaryRowData partition, int bucket, List<Tuple2<Long, Long>> kvs) throws Exception {
         Preconditions.checkNotNull(
                 service, "ExecutorService must be provided if writeFiles is needed");
         RecordWriter writer = createMergeTreeWriter(partition, bucket);
-        for (Tuple2<Integer, Integer> tuple2 : kvs) {
+        for (Tuple2<Long, Long> tuple2 : kvs) {
             writer.write(ValueKind.ADD, GenericRowData.of(tuple2.f0), GenericRowData.of(tuple2.f1));
         }
         List<SstFileMeta> files = writer.prepareCommit().newFiles();
         writer.close();
         return new ArrayList<>(files);
     }
 
-    private RecordWriter createMergeTreeWriter(BinaryRowData partition, int bucket) {
+    public RecordWriter createMergeTreeWriter(BinaryRowData partition, int bucket) {
```

Comment:
I have a question here. The accumulator is always initialized as a `DeduplicateAccumulator`(L98), can it be used in a value count test?

Reply:
Here is not to measure the underlying `merge`, so there will be no duplicate key appear, and this class will be useless.